dashboard/db/migrate/20151022174100_create_puzzle_ratings.rb

```diff
@@ -0,0 +1,15 @@
+class CreatePuzzleRatings < ActiveRecord::Migration
+  def change
+    create_table :puzzle_ratings do |t|
+      t.references :user, index: true
+      t.integer :script_id
+      t.integer :level_id
+      t.integer :rating
```

Comment:
the client code only allows values of 0 or 1. Should this also be enforced at the db or rails model level? What happens if someone submits a value that is not 0 or 1?